wavefront/common.go

```diff
@@ -15,6 +17,19 @@ func suppressSpaces(k, old, new string, d *schema.ResourceData) bool {
 	return strings.TrimSpace(old) == strings.TrimSpace(new)
 }
 
+func suppressJSONFormatting(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
I'm a bit confused by the name of this function. I believe it's just testing if the two json blobs can parse and are equal. Maybe something like `validAndEqual` or something?

Reply:
Correct. The idea behind this function is that it is comparing two JSON documents to see that they are semantically equal. That is they express the same thing even though they may have different tabs and whitespace in them.

Reply:
Named changed to isJSONForFieldTheSame.